### PR TITLE
fix: handle 77 level as it's first level

### PR DESCRIPTION
**Pull request by ishche**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by VitGottwald** [comment]
@ishche have you verified the behaviour against the IBM Enterprise COBOL compiler?

**Comment by ishche** [comment]
> @ishche have you verified the behaviour against the IBM Enterprise COBOL compiler?

No, I don't have access to it right now

File: server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/tree/logic/SectionNodeProcessorHelper.java [hunk 1]
```diff
@@ -237,22 +235,40 @@ private List<SyntaxError> processDefinition(
...
+                getParentForNextLevel(rootNode, lastVariableOnTheLevel, definitionNode),
+                definitionNode.getLevel(),
+                definitionNodes));
       }
     }
     return errors;
   }
 
+  private boolean isRootLevel(int level) {
+    return ImmutableSet.of(LEVEL_66, LEVEL_77, LEVEL_88, LEVEL_01).contains(level);
```
Comment (amanPras) [inline server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/tree/logic/SectionNodeProcessorHelper.java]: Not sure, but I guess 88 can be under a group item

Comment (ishche) [reply server/engine/src/main/java/org/eclipse/lsp/cobol/core/model/tree/logic/SectionNodeProcessorHelper.java]: That makes sense. There is a check for it before the new logic. I'll remove it from the list here.

**Review by amanPras — APPROVED** [review]

**Review by Nurkambay — APPROVED** [review]

**Pull request merged**